client-sdk/go/.golangci.yml

```diff
@@ -0,0 +1,73 @@
+# golangci-lint configuration.
```

Comment:
This seems more detailed than the one in oasis-core. Should we update the one in oasis-core as well?

Reply:
I actually took this one from oasis-core-ledger, but already needed to remove some lints. Yeah we should probably sync all of these across repositories.